NAS2D/Mixer/Mixer.h

```diff
@@ -23,7 +24,7 @@ namespace NAS2D
 	{
 	public:
 		static const int CONTINUOUS = -1;
-		static const int DEFAULT_FADE_TIME = 500;
+		static inline const std::chrono::milliseconds DEFAULT_FADE_TIME{500};
```

Comment:
Out of curiosity, have you tried:
```cpp
static constexpr std::chrono::milliseconds DEFAULT_FADE_TIME{500};
```

A `constexpr` is automatically `inline`. It would be rather appropriate to use here. Could also use `constexpr` on `CONTINUOUS` for consistency.

Reply:
I didn't think about that. I don't tend to use `constexpr` much due to its limited use cases.